### PR TITLE
modify eval func for labels with 2 unique values

### PR DESCRIPTION
**Pull request by double-y**
### Description
Support actual labels with 2 unique values

Fix https://github.com/exploratory-io/tam/issues/4679

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [x] Pass devtools::check()
- [x] Pass devtools::test()
- [ ] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/util.R [hunk 1]
```diff
@@ -705,3 +705,26 @@ pivot_ <- function(data, formula, value_col = NULL, fun.aggregate = mean, fill =
...
+#'convert values to binary label
+binary_label <- function(val) {
+  if(is.factor(val)){
+    # Need to subtract 1 because the first level in factor is regarded as 1
+    # though it should be FALSE.
+    val <- as.logical(as.integer(val) - 1)
+  } else {
+    logi_val <- as.logical(val)
+    # if the values are non-zero values,
+    # larger value should be regarrded as FALSE
```
Comment (hideaki) [inline R/util.R]: typo regarrded

File: R/util.R [hunk 2]
```diff
@@ -705,3 +705,26 @@ pivot_ <- function(data, formula, value_col = NULL, fun.aggregate = mean, fill =
...
+    if(all(logi_val[!is.na(logi_val)])){
```
Comment (hideaki) [inline R/util.R]: some more comments on the logic around here would help.

File: R/util.R [hunk 3]
```diff
@@ -705,3 +705,26 @@ pivot_ <- function(data, formula, value_col = NULL, fun.aggregate = mean, fill =
...
+      unique_val <- unique(val[!is.na(val)])
+      if(length(unique_val) == 2) {
+        logi_val <- val == max(unique_val)
+      } else if (length(unique_val) > 2) {
+        stop("actual labels can't have more than 2 unique values")
```
Comment (hideaki) [inline R/util.R]: To be a generic utility function, we should not assume that the column is called "actual".

Comment (double-y) [inline R/util.R]: modified to "binary labels can't have more than 2 unique values"

File: R/util.R (hunk 2 above)
Comment (double-y) [inline R/util.R]: Added comments

File: R/util.R (hunk 1 above)
Comment (double-y) [inline R/util.R]: Fixed it

**Pull request merged**